scipy/stats/_continuous_distns.py

```diff
@@ -3451,37 +3446,24 @@ def g(n):
         v = np.where(c < -0.5, np.nan, g1**2.0*gam2k)
 
         # skewness
-        def sk1_eval(x):
-            return _lazywhere(
-                x < -1./3, (x,), 
-                f=lambda y: np.full_like(y, np.nan), 
-                f2=lambda y: np.sign(y)*(-g3 + (g2 + 2*g2mg12)*g1)/g2mg12**1.5
-            )
-        
-        sk = _lazywhere(
-            abs(c) > eps**0.29, 
-            (c,),
-            f=sk1_eval, 
-            fillvalue=12*np.sqrt(6)*_ZETA3/np.pi**3
-        )
+        def sk1_eval(c, *args):
+            def sk1_eval_f(c, g1, g2, g3, g2mg12):
+                return np.sign(c)*(-g3 + (g2 + 2*g2mg12)*g1)/g2mg12**1.5
+            return _lazywhere(c >= -1./3, (c,)+args, f=sk1_eval_f, fillvalue=np.nan)
```

Comment:
Agains `>=` is simpler here. Also, we have to pass all the arguments into `_lazywhere` because they need to be the same shape as `c` when `c` is an array.